logstash-core/lib/logstash/environment.rb

```diff
@@ -15,7 +15,7 @@ module Environment
     end
   end
 
-  [
+  LOGSTASH_SETTINGS = [
```

Comment:
I may still use it in the specs, but I agree there are setters now for both config_ast and lscl. I'll try to revert the settings things.